### DISCUSSION TITLE
Recalculating Rates in kimai 2.X

### DISCUSSION THREAD

**Discussion by whizkid79** [kimai/kimai#4337] [Questions & Answers] [answered]
Hi,
I'm a newbie and made a mistake, when setting up the project for a customer by forgetting to set the proper rates in the project.
I just realised it now, as I wanted to export the data. But correcting it in the project, doesn't change the entries.
Now I have a month worth of time tracking entries which have the wrong (or no) price on it.

I found https://www.kimai.org/store/keleo-recalculate-rates-bundle.html mentioned in older issues, but it has a note, that it is no longer needed for 2.x installations. 

How can I trigger a recalculation for a whole project if a mistake was done? 

Thanks,
Peter

**Comment by kevinpapst** [comment, marked as answer]
Batch update mode.
- All Times > Search all entries > Checkbox on the upper left of the table
- Scroll down, click edit button under the table
- Follow to the "batch update screen"
- Choose "recalculate rates"
- Save

**Comment by whizkid79** [comment]
Thank you so much. If you know where to look, it's obvious.
